dev-libs/univalue/univalue-1.0.5.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools
+
+DESCRIPTION="C++ universal value object and JSON library"
+HOMEPAGE="https://github.com/jgarzik/univalue"
+SRC_URI="https://codeload.github.com/jgarzik/${PN}/tar.gz/v${PV} -> ${P}.tgz"
+
+LICENSE="MIT"
+SLOT="0/0"
+KEYWORDS="amd64 arm arm64 mips ppc ppc64 x86 amd64-linux x86-linux"
```

Comment:
You can't commit straight to stable. Repoman would've warned and stopped you here.